### PR TITLE
fix: Discord notification blocked because PR is from public fork

### PR DESCRIPTION
**Pull request by chubei**

### CODE REVIEW

**Comment by chubei** [comment]
I think `pull_request_review_comment` and `pull_request_review` will still fail for the same reason.

**Comment by v3g42** [comment]
@chubei Didnt seem to send a notification ?

**Comment by chubei** [comment]
> @chubei Didnt seem to send a notification ?

Need to merge and open a test PR to see

**Comment by v3g42** [comment]
Ah. right. this wouldnt trigger. Can you also change  `pull_request_review_comment` and `pull_request_review`

**Comment by chubei** [comment]
> Ah. right. this wouldnt trigger. Can you also change `pull_request_review_comment` and `pull_request_review`

No similar events for them like `pull_request_target` for `pull_request`. I'm looking for solutions.

File: .github/workflows/general.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Dozer General
 on:
   issues:
     types: [opened, edited, milestoned]
-  pull_request:
+  pull_request_target:
     branches: [main, release]
   pull_request_review_comment:
```
Comment (chubei) [inline .github/workflows/general.yaml:8]: Test comment

**Comment by chubei** [comment]
@v3g42 Let's merge this first so we can open test PR to see if it works

**Review by v3g42 — APPROVED** [review]

**Pull request merged**